src/semantic_world/robots.py

```diff
@@ -459,11 +459,13 @@ def from_world(cls, world: World) -> Self:
                                        thumb=left_gripper_thumb,
                                        finger=left_gripper_finger,
                                        _world=world)
-
-        left_arm = Arm(name=PrefixedName('left_arm', prefix=robot.name.name),
-                       root=world.get_body_by_name("l_shoulder_pan_link"),
-                       manipulator=left_gripper,
-                       _world=world)
+        left_arm = Arm(
+            name=PrefixedName("left_arm", prefix=robot.name.name),
+            root=world.get_body_by_name("l_shoulder_pan_link"),
```

Comment:
The root link of the arms is "torso_lift_link" because the l_shoulder_pan_joint is before the link